cpp/cppsdk/gsdk.cpp

```diff
@@ -154,7 +154,17 @@ namespace Microsoft
                 {
                     logFolder = "";
                 }
-
+#if _WIN32
```

Comment:
should this be just `#if WIN32`, similar to that? https://github.com/PlayFab/gsdk/blob/a329d1ba08a26ef8a122462212f10d98b0bd62c8/cpp/cppsdk/include/playfab/PlayFabBaseModel.h#L14

Reply:
it's also mentioned here as WIN32
https://github.com/PlayFab/gsdk/blob/master/cpp/CMakeLists.txt#L10

Reply:
upon some research _WIN32 is always defined, and WIN32 can also be not defined if no windows header is included, so I would stick with _WIN32

Reply:
nice, confirming that here! https://stackoverflow.com/a/662095 
Thanks!